src/lambda-helpers.ts

```diff
@@ -40,3 +42,32 @@ export const getGithubOpts = () => ({
     ).toString("base64")}`,
   },
 });
+
+// Twitter Creds
+const twitterConsumerKey = process.env.TWITTER_CONSUMER_KEY || "";
+const twitterConsumerSecret = process.env.TWITTER_CONSUMER_SECRET || "";
+
+export const getTwitterOpts = () => {
+  const twitterBearerTokenResponse = wrapAxios(
+      axios.get(
+      `https://api.twitter.com/oauth2/token`,
+      {
+        auth: {
+          username: twitterConsumerKey,
+          password: twitterConsumerSecret
+        },
+        params : {
+          grant_type: "client_credentials"
+        }
+      }
+    )
+  );
+
+  const twitterBearerToken = twitterBearerTokenResponse.body.access_token;
```

Comment:
this is really dope, great work! going to go add the twitter secrets now

Reply:
Ah, thanks! You were my inspiration!